people/people_service.go

```diff
@@ -225,6 +228,14 @@ func (s service) Count() (int, error) {
 	return results[0].Count, nil
 }
 
+func randSeq(n int) string {
```

Comment:
I thought maybe we could use the loop variable to get a character instead but after 15mins googling I am none the wiser how to do that but I did find that Go has "runes" which I find amusing and can't help but think they are mystical